include/suit_condition.h

```diff
@@ -34,5 +34,14 @@ int suit_condition_component_slot(struct suit_processor_state *state,
 int suit_condition_abort(struct suit_processor_state *state,
 		struct suit_manifest_params *component_params);
 
+#ifndef SUIT_PLATFORM_LEGACY_API_SUPPORT
+/** Check a manifest component integrtity and signature. */
```

Comment:
Small typo in integrtity.

Reply:
I'll let the CI run, then force-push :slightly_smiling_face: